agoric/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'Agoric'
+const endpoint = util.getEnv('AG_SOLO_ORACLE_URL')
```

Comment:
You could include this into `makeConfig`. No worry about extending the `Config` type

Reply:
Thanks, I had missed this in my self-review.